docs/source/contents/1_charge.rst

```diff
@@ -7,9 +7,9 @@ Membership
 ----------
 
 Meredith Hale, Lead
-Member
-Member
-Member
+Mark Baggett
+Bridger Dyson-Smith
+Mat Jordan
```

Comment:
Add returns between each name, otherwise they all run onto one line.

Reply:
Thanks for noting this - I didn't think about that. This has been addressed in my latest commit. I really appreciate your help.